bucket/vboxvmservice.json

```diff
@@ -0,0 +1,37 @@
+{
+    "version": "6.0-Pumpkin",
+    "homepage": "https://github.com/onlyfang/VBoxVmService/",
+    "description": "Windows Service to run VirtualBox VMs automatically",
+    "license": "https://github.com/onlyfang/VBoxVmService/blob/master/doc/Licence.txt",
+    "url": "https://github.com/onlyfang/VBoxVmService/releases/download/6.0-Pumpkin/VBoxVmService-6.0-Pumpkin.exe#/installer.exe",
+    "hash": "24cc6dcf0c5f8c25fca5181952e86c5525d83f981caece6c5de1f3e1ec673546",
+    "persist": "VBoxVmService.ini",
+    "bin": "VmServiceControl.exe",
+    "shortcuts": [
+        [
+            "VmServiceTray.exe",
+            "VmServiceTray"
+        ]
+    ],
+    "installer": {
+        "file": "installer.exe",
+        "args": [
+            "/VERYSILENT",
+            "/DIR=$dir"
+        ]
+    },
+    "uninstaller": {
+        "file": "unins000.exe",
+        "args": "/VERYSILENT"
+    },
+    "post_install": [
+        "Remove-Item \"$Env:USERPROFILE\\Desktop\\VmServiceTray.lnk\"",
+        "$d = \"$Env:ALLUSERSPROFILE\\Microsoft\\Windows\\Start Menu\\Programs\\VBoxVmService\"",
+        "Remove-Item \"$d\\VmServiceTray.lnk\"",
+        "if( (Get-ChildItem $d -Recurse -File | Measure-Object).Count -eq 0) { Remove-Item $d }"
+    ],
+    "checkver": "github",
```

Comment:
This cannot work

```suggestion
    "checkver": {
        "github": "https://github.com/onlyfang/VBoxVmService/",
        "regex": "/VBoxVmService-([\\w.-]+)\\.exe"
    },
```

Reply:
I pushed the manifest you had fixed. Looks cleaner :)

But I don't understand this change. The nvm example on [this page](https://github.com/lukesampson/scoop/wiki/App-Manifest-Autoupdate#special-cases) looks similar to the situation here so I thought it was alright.